sample_consensus/include/pcl/sample_consensus/sac.h

```diff
@@ -66,6 +66,8 @@ namespace pcl
       using Ptr = boost::shared_ptr<SampleConsensus<T> >;
       using ConstPtr = boost::shared_ptr<const SampleConsensus<T> >;
 
+      using IndicesPtr = boost::shared_ptr<std::vector<int>>;
```

Comment:
Is there a reason not to use our global `IndicesPtr` type here?

https://github.com/PointCloudLibrary/pcl/blob/65ecbd855ea96ffecac4fb32b4ab9224e1bc225c/common/include/pcl/pcl_base.h#L61

Reply:
Nice catch. I didn't check if the base class had anything!